k8s-scheduler/src/test/java/org/dcm/WorkloadGeneratorIT.java

```diff
@@ -71,31 +86,138 @@ public void testAffinityAntiAffinity() throws Exception {
 
         assertNotNull(schedulerName);
         LOG.info("Running testAffinityAntiAffinity with parameters: MasterUrl:{} SchedulerName:{}",
-                 fabricClient.getConfiguration().getMasterUrl(), schedulerName);
-
-        // Add a new one
-        final Deployment cacheExample = launchDeploymentFromFile("cache-example.yml", schedulerName);
-        final String cacheName = cacheExample.getMetadata().getName();
-        final Deployment webStoreExample = launchDeploymentFromFile("web-store-example.yml",
-                                                                    schedulerName);
-        final String webStoreName = webStoreExample.getMetadata().getName();
-
-        final int newPodsToCreate = cacheExample.getSpec().getReplicas() + webStoreExample.getSpec().getReplicas();
-        waitUntil((n) -> hasNRunningPods(newPodsToCreate));
-        final List<Pod> pods = fabricClient.pods().inNamespace(TEST_NAMESPACE).list().getItems();
-        assertEquals(newPodsToCreate, pods.size());
-
-        pods.forEach(pod -> assertNotEquals(pod.getSpec().getNodeName(), "kube-master"));
-
-        final Map<String, List<String>> podsByNode = new HashMap<>();
-
-        pods.forEach(pod -> podsByNode.computeIfAbsent(pod.getSpec().getNodeName(), k -> new ArrayList<>())
-                .add(pod.getMetadata().getName()));
-        podsByNode.forEach((nodeName, podsAssignedToNode) -> {
-            assertEquals(2, podsAssignedToNode.size());
-            assertTrue(podsAssignedToNode.stream().anyMatch(p -> p.contains(webStoreName)));
-            assertTrue(podsAssignedToNode.stream().anyMatch(p -> p.contains(cacheName)));
-        });
+                fabricClient.getConfiguration().getMasterUrl(), schedulerName);
+
+        // Load data from file
+        final ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        //final InputStream inStream = classLoader.getResourceAsStream("v1-data-start-time-sorted.txt");
+        final InputStream inStream = classLoader.getResourceAsStream("test-data.txt");
```

Comment:
Create one test method each that uses a different input file. That way, we can just invoke the specific trace we need from the commandline rather than having to recompile the code each time.

Reply:
Make sure to generate the appropriate error if the file can't be found: that is, ask the user to generate the traces from the workload-gen/ folder.